dictionary.json

```diff
@@ -1166,28 +1450,34 @@
     "fields": []
   },
   {
-    "toaq": "bo",
-    "type": "predicate",
-    "english": "▯ has/possesses ▯.",
-    "gloss": "have",
+    "toaq": "bo-",
+    "type": "prefix",
+    "english": "be X'd by …; passive",
+    "gloss": "PASS",
     "short": "",
     "keywords": [],
-    "frame": "c e",
-    "distribution": "d d",
-    "pronominal_class": "ho",
     "notes": [],
-    "examples": [],
-    "fields": []
+    "examples": []
   },
   {
-    "toaq": "bodoa",
+    "toaq": "boju",
+    "type": "interjection",
+    "english": "requesting to take the floor; may I speak?",
+    "gloss": "",
+    "short": "",
+    "keywords": [],
+    "notes": [],
+    "examples": []
+  },
+  {
+    "toaq": "bo",
     "type": "predicate",
-    "english": "▯ gives ▯ possession of ▯.",
-    "gloss": "give",
+    "english": "▯ has/possesses ▯.",
+    "gloss": "have",
     "short": "",
     "keywords": [],
-    "frame": "",
-    "distribution": "d d d",
+    "frame": "c e",
```

Comment:
`a` for an evidently transitive verb feels wrong. Instead, since not only genitival but also attributive serialization doesn't exist, I'd suggest moving `c e` to `c c` and `a` to `c`, making `frame` fully semantic and no longer serial-behavior-based.

Reply:
mmm, that'd be quite a big change. Instead could we just replace all `"frame": "c e"` and `"frame": "a"` with `"frame": ""` for the moment, and decide on that later? Otherwise that adds a whole new layer of stuff to review here.

Reply:
In that case I'd prefer to leave frame data unchanged and open a separate ticket to discuss the matter there, to keep concerns separated. Straight-out removing most frames feels unsatisfying and might come at a disadvantage when we have to add those frames back later.

Reply:
OK, works for me

Reply:
Opening #151 while resolving this.